payloads/start.go

```diff
@@ -242,6 +242,10 @@ type StartCmd struct {
 	// Storage contains all the information required to attach or boot
 	// from storage for the new instance.
 	Storage []StorageResource `yaml:"storage,omitempty"`
+
+	// Migration is set to true if the payload represents a request to
+	// restart an existing instance on a new node.
+	Migration bool
```

Comment:
And further could this be resolved by tracking the state in the the controller database rather than relaying information back from launcher that controller told it in the first place?